.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,60 @@
+---
+# This GitHub Actions workflow is calculating code coverage of extension and creates badge out of it.
+name: Code coverage
+on: push
+jobs:
+  code-coverage:
+    name: "Code coverage check"
+    runs-on: ubuntu-18.04
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
```

Comment:
I propose we keep this out of this PR and move it to a separate PR so we can have it for the other checks as well.